pacstall

```diff
@@ -25,14 +25,17 @@
 # Configuration
 export LC_ALL=C
 export LOGDIR="/var/log/pacstall/metadata"
-export LOGFILE="/var/log/pacstall/error_log/$(date +"%F_%T.log")"
+LOGFILE="/var/log/pacstall/error_log/$(date +"%F_%T.log")"
+export LOGFILE
```

Comment:
????

Reply:
Fixes https://github.com/koalaman/shellcheck/wiki/SC2155

Reply:
It's better for future tests, and the CI

Reply:
> Fixes https://github.com/koalaman/shellcheck/wiki/SC2155

Got it